wiki/Article_styling_criteria/Formatting/ko.md

```diff
@@ -0,0 +1,903 @@
+HTML 주석은 해야 할 작업을 표시하는 데 사용되지만, 텍스트에 설명글을 다는데도 사용할 수 있습니다. 주석은 각 줄에 있어야 하지만, 문단의 줄 내부에도 배치할 수도 있습니다. 줄 내부에 배치를 할 경우, 주석의 앞부분에는 공백을 두어선 안됩니다.
+
+안 좋은 예:
+
+```markdown
+HTML 주석은 <!-- TODO explain HTML comments --> 할 일을 표시하거나 텍스트에 주석을 달기 위해 사용해야 합니다.
+```
+
+좋은 예:
+
+```markdown
+HTML 주석은<!-- TODO explain HTML comments --> 할 일을 표시하거나 텍스트에 주석을 달기 위해 사용해야 합니다.
+```
+
+## 수정
+
+### 라인 종단 부호
+
+*주의: GitHub을 이용해 `CRLF` (캐리지 리턴 및 라인 피드)를 사용한 Markdown 파일을 업로드하는 경우 해당 파일이 `CRLF`를 사용하게 됩니다. 이를 막기 위해, 업로드하기 전 라인의 끝을 `LF` (라인 피드)로 설정해 주세요.*
+
+Markdown 파일의 라인 종단 부호가 `LF`인지 확인하세요.
+
+### 이스케이프
+
+Markdown 문법은 필요에 따라 이스케이프 해야 하지만, 문서의 제목은 일반 텍스트로 처리되므로 이스케이프를 하면 안 됩니다.
+
+### 문단
+
+각 문단 다음에는 반드시 빈 줄 하나가 따라와야 합니다.
+
+### 줄 바꿈
+
+줄 바꿈은 백슬래시 (`\`)를 사용해야 합니다.
+
+줄 바꿈은 드물게 사용해야 합니다.
+
+## 햇노트
+
+*[공지](#공지)와 헷갈리지 마세요.*
+
+햇노트는 관련 문서로 유도하거나 관련 주제에 대해 알리는 데 도움을 주기 위해 문서 혹은 섹션의 맨 위에 배치되는 짧은 노트입니다.
+
+햇노트는 기울임체로 표시해야 하며 제목 바로 다음에 배치해야 합니다. 여러개의 햇노트를 사용하는 경우, 같은 단락에 줄 바꿈으로 구분하여 입력해야 합니다.
+
+### 메인 페이지
+
+*메인 페이지* 햇노트는 주제의 메인 문서를 나타냅니다. 이 햇노트를 사용한다는 것은 해당 섹션이 링크된 페이지에 대한 요약 정보임을 의미합니다. 이 햇노트는 반드시 하나의 링크만을 가지고 있어야 합니다. 다음과 같은 형식으로 사용하세요:
+
+```markdown
+*메인 페이지: {문서}*
+
+*메인 페이지: {문서} 및 {문서}*
+```
+
+### 또한 참조하세요
+
+*또한 참조하세요* 햇노트는 주어진 문서 혹은 섹션에서 관심을 가져볼 만한 다른 포인트를 제시합니다. 다음과 같은 형식으로 사용하세요:
+
+```markdown
+*또한 {문서}도 참조하세요.*
+
+*또한 {문서}와 {문서}도 참조하세요.*
+```
+
+### 해당된다면 참조하세요
+
+*해당된다면 참조하세요* 햇노트는 *또한 참조하세요* 햇노트와 비슷하지만, 일반적으로 더 직접적으로 설명합니다. 이 햇노트는 필요에 따라 2개 이상의 링크를 사용할 수 있습니다. 다음과 같은 형식으로 사용하세요:
+
+```markdown
+*{내용}이라면 {문서}를 참조하세요.*
+
+*{내용}이라면 {문서} 및 {문서}를 참조하세요.*
+```
+
+### 헷갈리지 마세요
+
+*헷갈리지 마세요* 햇노트는 헷갈릴 수 있거나 잘못 이해할 수 있는 문서 제목 혹은 섹션을 구별하는 데 도움을 줍니다. 이 햇노트는 필요에 따라 2개 이상의 링크를 사용할 수 있습니다. 다음과 같은 사용하세요:
+
+```markdown
+*{문서}와 헷갈리지 마세요.*
+
+*{문서} 혹은 {문서}와 헷갈리지 마세요.*
+```
+
+### 다르게도 사용할 수 있습니다
+
+*다르게도 사용할 수 있습니다* 햇노트는 *헷갈리지 마세요* 햇노트와 비슷하지만, [동음이의어 문서](#동음이의어-문서)에 직접 연결됩니다. 이 햇노트의 링크는 동음이의어 문서에만 연결되어야 합니다. 다음과 같은 형식으로 사용하세요:
+
+```markdown
+*{동음이의어 문서}로도 사용될 수 있습니다.*
+```
+
+## 공지
+
+*[햇노트](#햇노트)와 헷갈리지 마세요.*
+
+공지는 해당되는 섹션에 배치되지만, 단락의 시작에 사용되어야 하며 기울임체를 사용해야 합니다. 적절한 경우에는 굵게 표시할 수 있지만 최소한으로 적용해야 하며, 완전한 문장으로 작성되어야 합니다. 따라서, 대부분의 [햇노트](#햇노트)와는 달리 마침표 (`.`) 또는 느낌표 (`!`)를 적절하게 사용해야 합니다. 공지와 같은 단락에 있는 모든 것들은 기울임체로 작성되어야 합니다. 다음과 같은 형식으로 사용하세요:
+
+```markdown
+*참고: {참고}.*
+
+*알림: {알림}.*
+
+*주의: {주의}.*
+
+*경고: {경고}.*
+```
+
+- `참고`은 사실이나 사소한 내용에 사용합니다.
+- `알림`는 알아야 할 내용을 상기시거나 관심을 가지게 할 때 사용합니다.
+- `주의`는 의도하지 않은 결과가 발생하지 않도록 경고하는 데 사용합니다.
+- `경고`는 변경이 될 수 있음을 경고하는 데 사용해야 합니다.
+
+## 햇노트 및 공지 스택
+
+필요한 경우 여러개의 햇노트 혹은 공지를 스택 할 수 있습니다. 이 경우 공백 라인이 없도록 스택하고 백슬래시가 뒤에 따라와야 합니다:
+
+```markdown
+*경고: {경고}.*\
+*또한 {문서}도 참조하세요.*
+```
+
+대부분의 경우에는 문단에 관련 없는 햇노트나 공지를 여러개 스택하는 것보다 문단 텍스트에 포함시키는 것이 더 적합할 수 있습니다.
+
+## 강조
+
+### 굵게
+
+굵은 텍스트는 이중 별표 (`**`)를 사용해야 합니다.
+
+단락의 첫 부분은 문서 제목의 첫번째 항목을 굵게 표시할 수 있습니다.
+
+### 기울임체
+
+기울임체은 단일 별표 (`*`)를 사용해야 합니다.
+
+축약형이나 두문자어, 혹은 이니셜이 처음 나오는 경우 기울임체로 표시할 수 있습니다.
+
+기울임체는 강조하거나 가독성의 개선을 위해 사용될 수도 있습니다.
+
+작품이나 비디오 게임의 이름은 기울임체로 표기해야 합니다. osu! — 게임 —은 예외입니다.
```

Comment:
shouldn't it be like this? (if em dashes are even a thing in korean)

```suggestion
작품이나 비디오 게임의 이름은 기울임체로 표기해야 합니다. osu! — 게임 — 은 예외입니다.
```

Reply:
I agree with yours. probably it's a mistake.

Reply:
> I agree with yours. probably it's a mistake.

I realized that it's the correct grammar in Korean after I talked with the author in the private messages about this.

reference: https://kornorms.korean.go.kr/regltn/regltnView.do#a770 (from National Institute of Korean Language)